### PR TITLE
Blog post for CSV activation

### PR DESCRIPTION
**Pull request by btcdrak**
Written by @jl2012 and submitted by me on request

ping @harding

### CODE REVIEW

File: _posts/en/posts/2016-06-21-csv-softfork-instructions.md [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+---
+{% include _toc.html %}
+
+There is an ongoing soft fork of the Bitcoin consensus rules. While everything appears to be proceeding well, this article contains important information and checklists for miners and pool operators which must not be ignored.
+
+If there is any doubt, miners and pool operators are welcome to [contact us](https://bitcoincore.org/en/contact/).
+
+TL;DR 
+1. Check all your nodes have been upgraded to Bitcoin Core 0.12.1 or compatible software. This must happen before block #419328.
+2. If you manually hard code the block version field, or use the nSqeuence or nLockTime fields in the coinbase transaction, you must follow the instructions carefully.
```
Comment (paveljanik) [inline _posts/en/posts/2016-06-21-csv-softfork-instructions.md]: nSequence

Comment (btcdrak) [inline _posts/en/posts/2016-06-21-csv-softfork-instructions.md]: fixed

**Comment by jonasschnelli** [comment]
ACK e9986d7dec77d3bdad8cd0be8dcc1eb40abacd7c

**Comment by laanwj** [comment]
ACK

**Comment by jl2012** [comment]
thanks for editing. ACK

**Comment by btcdrak** [comment]
OK I'm going to publish this since the message needs to get out sooner than later.

**Comment by laanwj** [comment]
Agree

**Comment by petertodd** [comment]
ACK

**Pull request merged**